### PR TITLE
Console release notes 2022-12-01

### PR DESCRIPTION
**Pull request by nikitakalyanov**

### CODE REVIEW

File: content/release-notes/2022-12-01.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- Control plane: List operations in V2 API supports cursor pagination.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:7]:
```suggestion
- API: List operations in V2 API supports cursor pagination.
```

File: content/release-notes/2022-12-01.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Added validation for empty names for branches and
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:8]:
```suggestion
- API: Added validation for empty names for branches and 
```

File: content/release-notes/2022-12-01.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- Control plane: List operations in V2 API supports cursor pagination.
+- Control plane: Added validation for empty names for branches and 
+projects. 
+- Control plane: Added LSN validation.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:10]:
```suggestion
- API: Added LSN validation.
```

File: content/release-notes/2022-12-01.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- Console: Added 'delete endpoint' button.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:11]:
```suggestion
- UI: Added 'delete endpoint' button.
```

File: content/release-notes/2022-12-01.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Make parent_id and name optional when creating a branch.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:12]:
```suggestion
- API: Make parent_id and name optional when creating a branch.
```

File: content/release-notes/2022-12-01.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- Console: Show branch_id and fork button in branch view.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:13]:
```suggestion
- UI: Show branch_id and fork button in branch view.
```

File: content/release-notes/2022-12-01.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Added new autoscaling limits.
+- Control plane: Added region maintenance/testing flag.
+- Control plane: Return complete host for connecting in endpoint response.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:16]:
```suggestion
- API: Return complete host for connecting in endpoint response.
```

File: content/release-notes/2022-12-01.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Rename default database from `main` to `neondb`.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:17]:
```suggestion
- Control plane: Rename the database that is created along with the project from `main` to `neondb`.
```

File: content/release-notes/2022-12-01.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Add per-project branch/endpoint limits.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:18]:
```suggestion
- UI: Add per-project branch/endpoint limits.
```

not sure about this one, maybe we should call out the update of the free tier here instead.

File: content/release-notes/2022-12-01.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Support allow_connections flag for an endpoint.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:19]:
```suggestion
- API: Support allow_connections flag for an endpoint.
```

File: content/release-notes/2022-12-01.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+- Console: Allow to choose an endpoint in passwordless auth.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:20]:
```suggestion
- UI: Allow to choose an endpoint in passwordless auth.
```

File: content/release-notes/2022-12-01.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Added project maintenance flag.
+- Control plane: Allow to change the branch of an endpoint.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:22]:
```suggestion
- API: Allow to change the branch of an endpoint.
```

File: content/release-notes/2022-12-01.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Allow creation of a branch without an endpoint.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:23]:
```suggestion
- API: Allow creation of a branch without an endpoint.
```

File: content/release-notes/2022-12-01.md [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+- Control plane: Creating a branch with V1 is now deprecated.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:24]:
```suggestion
- API: Creating a branch with V1 is now deprecated.
```

File: content/release-notes/2022-12-01.md [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+- Console: Added API response examples.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:25]:
```suggestion
- API: Added response examples to the API Spec.
```

File: content/release-notes/2022-12-01.md [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+
+### Bug fixes
+
+- UI: Fixed rerender during endpoint polling.
+- UI: Stop polling on project deletion.
+- Control Plane: Return correct error on duplicate branch name.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:31]:
```suggestion
- API: Return correct error on duplicate branch name.
```

File: content/release-notes/2022-12-01.md [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+- UI: Handle errors when polling endpoint status.
+- Control plane: Handle ID collisions when creating a branch or endpoint.
+- Control plane: Handle repeated endpoint suspension.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:34]:
```suggestion
- API: Handle repeated endpoint suspension.
```

not sure about this one

File: content/release-notes/2022-12-01.md [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+- UI: don't show internal service roles.
+- Control plane: fix ordering of operations and projects.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:36]:
```suggestion
- API: fix ordering of operations and projects.
```

File: content/release-notes/2022-12-01.md [hunk 19]
```diff
@@ -0,0 +1,38 @@
...
+- UI: Handle sign-in during passwordless authentication.
```
Comment (stepashka) [inline content/release-notes/2022-12-01.md:37]:
```suggestion
- UI: Handle sign-in during passwordless connect.
```

**Review by stepashka — COMMENTED** [review]
Have we changed the API v2 region id logic in this release? Please elaborate on that for the users who may depend on this

**Review by danieltprice — APPROVED** [review]
approve for merge

File: content/release-notes/2022-12-01.md (hunk 1 above)
Comment (danieltprice) [reply content/release-notes/2022-12-01.md]: fixed.

File: content/release-notes/2022-12-01.md (hunk 2 above)
Comment (danieltprice) [reply content/release-notes/2022-12-01.md]: fixed.

**Pull request merged**